### ISSUE TITLE
passing without else statements

### ISSUE THREAD

**Issue by hathcode** [freeCodeCamp/freeCodeCamp#13678]
Challenge [Logical Order in If Else Statements](https://www.freecodecamp.com/challenges/logical-order-in-if-else-statements#?solution=%0Afunction%20orderMyLogic(val)%20%7B%0A%20%20if%20(val%20%3C%205)%20%7B%0A%20%20%20%20return%20%22Less%20than%205%22%3B%0A%20%20%7D%20%0A%20%20%0A%20%20if%20(val%20%3C%2010)%20%7B%0A%20%20%20%20return%20%22Less%20than%2010%22%3B%0A%20%20%7D%20%0A%0A%20%20%20%20return%20%22Greater%20than%20or%20equal%20to%2010%22%3B%0A%20%20%0A%7D%0A%0A%2F%2F%20Change%20this%20value%20to%20test%0AorderMyLogic(7)%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript

function orderMyLogic(val) {
  if (val < 5) {
    return "Less than 5";
  } 
  
  if (val < 10) {
    return "Less than 10";
  } 

    return "Greater than or equal to 10";
  
}

// Change this value to test
orderMyLogic(7);

```

Passing tests without the desired else statements.

**Comment by Ethan-Arrowood** [comment]
@hathcode This isn't a github issue. Please discuss this on the Gitter or Forum. If you think something else is wrong feel free to elaborate and we can discuss further.

**Comment by Greenheart** [comment]
@hathcode Thanks for reporting this!

As the `else` block can be removed when a function involves `return`s, this is valid code.

The challenge doesn't ask for the `else` block specifically, it only asks for the right functionality - which this code fulfills.

I'm closing this for now, but please add comments if you have further feedback.

Happy coding! :blush:

**Comment by hathcode** [comment]
I'm not asking for help, Maybe I should have clarified further in the post.  

Perhaps I inferred something from the directions I should not have.  As I understood it when I completed the challenge, else statements were required.  If that was the case, then a case passing without else cases is a bug, hence the posted issue.

On second look though, maybe all they wanted in the challenge was to reorder the statements, nothing less, nothing more.  If that is the case, there is no issue.  I guess I was just in "if/else" mode from the previous challenges!

My bad :-)
